scripts/ci/setup-dev-sso.sh

```diff
@@ -0,0 +1,289 @@
+    INSTALL_KEYCLOAK="apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: sandbox-dev
+  namespace: ${DEV_SSO_NS}
+  labels:
+    sso-toolchain: sandbox-dev
+spec:
+  externalAccess:
+    enabled: true
+  instances: 1
+---
+ apiVersion: keycloak.org/v1alpha1
+ kind: KeycloakRealm
+ metadata:
+   name: sandbox-dev
+   namespace: ${DEV_SSO_NS}
+ spec:
+   instanceSelector:
+     matchLabels:
+       sso-toolchain: sandbox-dev
+   realm:
+     id: sandbox-dev
+     realm: sandbox-dev
+     displayName: Sandbox Dev In-cluster Keycloak
+     accessTokenLifespan: 7200
+     accessTokenLifespanForImplicitFlow: 900
+     enabled: true
+     sslRequired: none
+     registrationAllowed: false
+     registrationEmailAsUsername: false
+     rememberMe: false
+     verifyEmail: false
+     loginWithEmailAllowed: true
+     duplicateEmailsAllowed: false
+     resetPasswordAllowed: false
+     editUsernameAllowed: false
+     bruteForceProtected: false
+     permanentLockout: false
+     maxFailureWaitSeconds: 900
+     minimumQuickLoginWaitSeconds: 60
+     waitIncrementSeconds: 60
+     quickLoginCheckMilliSeconds: 1000
+     maxDeltaTimeSeconds: 43200
+     failureFactor: 30
+     clients:
+       - id: 9a5018a7-5f92-40c9-b8f1-63f53bc32a68
+         clientId: sandbox-public
+         surrogateAuthRequired: false
+         enabled: true
+         clientAuthenticatorType: client-secret
+         redirectUris:
+           - '*'
+         webOrigins:
+           - '*'
+         notBefore: 0
+         bearerOnly: false
+         consentRequired: false
+         standardFlowEnabled: true
+         implicitFlowEnabled: false
+         directAccessGrantsEnabled: true
+         serviceAccountsEnabled: false
+         publicClient: true
+         frontchannelLogout: false
+         protocol: openid-connect
+         protocolMappers: []
+         attributes: {}
+         authenticationFlowBindingOverrides: {}
+         fullScopeAllowed: true
+         nodeReRegistrationTimeout: -1
+         defaultClientScopes: []
+         optionalClientScopes: []
+     clientScopes: []
+     defaultDefaultClientScopes: []
+     smtpServer: {}
+     loginTheme: rh-sso
+     eventsEnabled: false
+     eventsListeners:
+       - jboss-logging
+     enabledEventTypes: []
+     adminEventsEnabled: false
+     adminEventsDetailsEnabled: false
+     identityProviders: []
+     identityProviderMappers: []
+     internationalizationEnabled: false
+     supportedLocales: []
+     authenticationFlows: []
+     authenticatorConfig: []
+     userManagedAccessAllowed: false
+     users:
+       - credentials:
+           - type: password
+             value: user1
+         email: user1@user.us
+         emailVerified: true
+         enabled: true
+         firstName: user1
+         id: user1
+         username: user1
+         clientRoles: {}"
+    echo "objects to be created in order to install operator"
+    cat <<EOF | oc apply -f -
+${INSTALL_KEYCLOAK}
+EOF
```

Comment:
ditto

Reply:
changed in https://github.com/codeready-toolchain/toolchain-cicd/pull/113/commits/07761694b9f83d5e089c2a366f10a487e7934216